inc/cachify_db.class.php

```diff
@@ -131,7 +132,7 @@ public static function print_cache( $cache ) {
 
 		/* Signature */
 		if ( isset( $cache['meta'] ) ) {
-			echo self::_cache_signature( $cache['meta'] );
+			echo self::_cache_signature( $sigDetail, $cache['meta'] );
```

Comment:
What is the reason to store the signature in APC/HDD/Memcache in the cached page itself while DB is generating the signature during the output process? I think, this would make sense to unify.